File: front-end/src/response/ExamHistoryResponse.ts
import AccountResponse from "./AccountResponse";
import ExamDetailsResponse from "./details/ExamDetailsResponse";

export default interface ExamHistoryResponse{
    id: number;
    point: number;
    correct_count:number;
    exam: ExamDetailsResponse;
    account: AccountResponse;
    end_time: Date;
    created_date: Date;
    last_modified_date?: Date;
}

File: back-end/src/main/java/com/ta2khu75/quiz/entity/response/ExamHistoryResponse.java
package com.ta2khu75.quiz.entity.response;

import java.time.LocalDateTime;

import com.ta2khu75.quiz.entity.response.details.ExamDetailsResponse;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
@Data
@NoArgsConstructor
@AllArgsConstructor
public class ExamHistoryResponse {
	Long id;
	float point;
	int correctCount;
	ExamDetailsResponse exam;
	AccountResponse account;
	LocalDateTime endTime;
	LocalDateTime createdDate;
	LocalDateTime lastModifiedDate;
}
